tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.md

```diff
@@ -181,7 +181,23 @@ This test requires that a vSphere server is running and available
 8. Docker container created with network attached with net1, got ip address inside of network range
 9. Docker create another container, and link to previous one, can talk to the the first container successfully
 
-
+## Container Firewalls
+1. Create an open container and verify another open container can connect to it on arbitrary ports.
+2. a. Try to publish a port on a closed firewall and verify an error is received.
+   b. Create a closed container and verify an open container cannot connect to it on an arbitrary port.
+   c. Create a container connected to a bridge and a closed network. Verify that another container connected
+      to the same bridge can connect to the closed container.
+3. a. Create an outbound container. Verify that an outbound container on the same external network cannot 
```

Comment:
3c. verify that two outbound containers on the same external network and on the same bridge network _can_ talk to one another via the bridge (confirms that routing/name resolution are working appropriately). We may need to use the fully qualified `containerA.bridgenet` name if the external network is the default.